x-pack/plugin/runtime-fields/src/main/java/org/elasticsearch/xpack/runtimefields/mapper/ScriptIpMappedFieldType.java

```diff
@@ -119,14 +109,18 @@ public Query rangeQuery(
     public Query termQuery(Object value, QueryShardContext context) {
         checkAllowExpensiveQueries(context);
         if (value instanceof InetAddress) {
-            return InetAddressPoint.newExactQuery(name(), (InetAddress) value);
+            return inetAddressQuery((InetAddress) value, context);
```

Comment:
This was a bug!

Reply:
should we push this separately?

Reply:
I'm ok pushing it as part of this because this is ready.